### PR TITLE
Cleanup verbose drop event logging

### PR DESCRIPTION
**Pull request by shastrinator**
Signed-off-by: Kiran Shastri <shastrinator@gmail.com>

### CODE REVIEW

File: pkg/hostagent/event_poster.go [hunk 1]
```diff
@@ -113,9 +113,9 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	agent.indexMutex.Lock()
 	srcPodKey, srcOk := agent.podIpToName[packetEvent.SourceIP]
 	agent.indexMutex.Unlock()
-	agent.log.Debug("srcPodKey for ", packetEvent.SourceIP, ": ", srcPodKey)
+	agent.log.Trace("srcPodKey for ", packetEvent.SourceIP, ": ", srcPodKey)
```
Comment (tomflynn) [inline pkg/hostagent/event_poster.go:116]: should this be below in the else...it doesn't seem useful if !srcOk

File: pkg/hostagent/event_poster.go [hunk 2]
```diff
@@ -143,9 +143,9 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	agent.indexMutex.Lock()
 	dstPodKey, dstOk := agent.podIpToName[packetEvent.DestinationIP]
 	agent.indexMutex.Unlock()
-	agent.log.Debug("dstPodKey for ", packetEvent.DestinationIP, ": ", dstPodKey)
+	agent.log.Trace("dstPodKey for ", packetEvent.DestinationIP, ": ", dstPodKey)
```
Comment (tomflynn) [inline pkg/hostagent/event_poster.go:146]: same comment here

File: pkg/hostagent/event_poster.go (hunk 1 above)
Comment (shastrinator) [reply pkg/hostagent/event_poster.go:116]: Agreed. Will change.

File: pkg/hostagent/event_poster.go (hunk 2 above)
Comment (shastrinator) [reply pkg/hostagent/event_poster.go:146]: Agreed. Will change.

**Pull request merged**